monitoring/grafana/KubeStellarControllersMonitoring.json

```diff
@@ -0,0 +1,837 @@
+                "readOnly": true
+              }
+            },
+            "properties": [
+              {
+                "id": "custom.hideFrom",
+                "value": {
+                  "legend": false,
+                  "tooltip": false,
+                  "viz": true
+                }
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 0
+      },
+      "id": 1,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus-test"
+          },
+          "disableTextWrap": false,
+          "editorMode": "code",
+          "exemplar": false,
+          "expr": "histogram_quantile(0.99, sum by(instance, name, le) (rate(workqueue_queue_duration_seconds_bucket{namespace=\"$SpaceNS\", job=\"$KSController\"}[5m])))",
+          "fullMetaSearch": false,
+          "includeNullMetadata": true,
+          "instant": false,
+          "interval": "",
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "A",
+          "useBackend": false
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus-test"
+          },
+          "editorMode": "code",
+          "expr": "histogram_quantile(0.50, sum by(instance, name, le) (rate(workqueue_queue_duration_seconds_bucket{namespace=\"$SpaceNS\", job=\"$KSController\"}[5m])))",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "B"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus-test"
+          },
+          "editorMode": "code",
+          "expr": "histogram_quantile(0.25, sum by(instance, name, le) (rate(workqueue_queue_duration_seconds_bucket{namespace=\"$SpaceNS\", job=\"$KSController\"}[5m])))",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "C"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "prometheus-test"
+          },
+          "editorMode": "code",
+          "expr": "histogram_quantile(0.75, sum by(instance, name, le) (rate(workqueue_queue_duration_seconds_bucket{namespace=\"$SpaceNS\", job=\"$KSController\"}[5m])))",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "D"
+        }
+      ],
+      "title": "Percentile of Sync Duration for Items in the Workqueue",
```

Comment:
This graph starts out with all the traces deselected. See the blank graph in the attached screenshot. Traces appear after I click on an entry in the legend table. But then, I can not individually enable and disable them.

<img width="452" alt="Screenshot 2024-08-13 at 4 43 01 PM" src="https://github.com/user-attachments/assets/c3058001-8455-4bf2-9195-05bd7daef2d3">

Reply:
@MikeSpreitzer, I never seen that before unless when starting with a fresh environment as it will take several seconds for the graph to get populated. I'll make a note of this and push a fix in a new PR if needed.

Reply:
It is not about data availability. As soon as I click on an entry in the legend table, data appears.

The reason that the traces cannot be enabled/disabled individually is that they have the exact same legend. Putting the percentile number in the legend fixes that problem.

Reply:
Fixed! Using the steps provided in the screenshot in the below comment

Reply:
I am testing the latest revision, now commit 988e4691ea5e1a1b54209d0ab679b1eadee963ea . This panel is still starting with all the traces disabled (although they now have distinct names).

<img width="446" alt="Screenshot 2024-08-13 at 10 37 26 PM" src="https://github.com/user-attachments/assets/f60b1849-2c09-4664-9d10-5a322bff178a">